src/web100srv.c

```diff
@@ -2339,49 +2185,36 @@ void NDT_server_main_loop(SSL_CTX *ssl_context, int listenfd) {
  * @param argc Number of arguments
  * @param argv string command line arguments
  * */
-int main(int argc, char** argv) {
-  pid_t chld_pid = -1;
-  int retcode;
-  int tpid, mwaiting = 0;
-  int ctlsockfd = -1;
-  int c, chld_pipe[2];
-  int i, loopcnt, t_opts = 0;
-  struct sockaddr_storage cli_addr;
+int main(int argc, char **argv) {
+  int c, i, loopcnt;
   struct sigaction new;
-  tcp_stat_agent* agent;
   char *lbuf = NULL, *ctime();
-  char buff[32], tmpstr[256];
-  char test_suite[16];
-  FILE * fp;
+  FILE *fp;
   size_t lbuf_max = 0;
-  fd_set rfd;
-  struct timeval sel_tv;
-  struct ndtchild *tmp_ptr = NULL, *new_child = NULL, *mchild = NULL;
+  char *private_key_file = NULL;
+  char *certificate_file = NULL;
+  SSL_CTX *ssl_context = NULL;
+  char ssl_error[255];
+  unsigned long ssl_err;
```

Comment:
Can we rename to `ssl_error` -> `ssl_error_msg` and `ssl_err` -> `ssl_error`? I think having `ssl_error` and `ssl_err` together in the same function is confusing.